# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/EarlyHintPreloader.cpp
# Commit: 8575cd92e1d8
# Full Hash: 8575cd92e1d8cd783e6cca3f6feadef11f0c4cc8
# Author: Manuel Bucher <manuel@mozilla.com>
# Date: 2022-12-03 09:24:59
# Regressor Bug: 1771867
# File Overlap Count: 1
# Description:
#   Bug 1771867 - Early Hints Phase 2 - Part 8: Add list of open channels from OngoingEarlyHints to RedirectToRealChannelArgs r=necko-reviewers,valentin,kershaw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D161179
# ==============================================================================

diff -r ce8f0a8fa688 -r 8575cd92e1d8 netwerk/protocol/http/EarlyHintPreloader.cpp
--- a/netwerk/protocol/http/EarlyHintPreloader.cpp	Fri Dec 02 16:15:55 2022 +0000
+++ b/netwerk/protocol/http/EarlyHintPreloader.cpp	Fri Dec 02 16:15:56 2022 +0000
@@ -66,6 +66,11 @@
   return mOngoingPreloads.InsertOrUpdate(aKey, aPreloader);
 }
 
+void OngoingEarlyHints::RegisterLinksAndGetConnectArgs(
+    nsTArray<EarlyHintConnectArgs>& aOutLinks) {
+  // register all channels before returning
+}
+
 //=============================================================================
 // EarlyHintPreloader
 //=============================================================================